test/tbb/test_try_put_and_wait.h

```diff
@@ -0,0 +1,468 @@
+/*
```

Comment:
Most part of this file was moved from test_buffering_try_put_and_wait without any changes